content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+---
+title: "Contact Routing Guide"
+linkTitle: "Contact Routing Guide"
+description: "Set up a Sensu Enterprise Contact Routing."
+product: "Sensu Enterprise"
+version: "2.6"
+weight: 7
+menu:
+ sensu-enterprise-2.6:
+   parent: guides
+---
+
+- [Prerequisites](#prerequisites)
+- [Overview](#overview)
+  - [Contact Routing Basics](#contact-routing-basics)
+    - [Contact Configuration](#contact-configuration)
+    - [Check Configuration](#check-configuration)
+    - [Client Configuration](#client-configuration)
+- [Example Implementation](#example-implementation)
```

Comment:
Needs to be `example-implementations` or the heading below needs to be changed to `example-implementation` as this link doesn't work.

Reply:
updated the header for section to be plural